ui.html

```diff
@@ -118,4 +140,6 @@
   <button onclick='debuginfo()' id='debuginfo'>debuginfo</button> (see console)
   <br>
   <button onclick='debugSend()' id='debugtest'>debug send label 'snooze_test'</button>
+  <br>
```

Comment:
I think if we want these on separate lines, putting them in separate `<div>`s is a better (more semantic) approach, but not a big deal if it works.